pages/index.tsx

```diff
@@ -1,4 +1,5 @@
 import Link from 'next/link';
+import Head from 'next/head';
```

Comment:
Is it being used?

Reply:
Ops, I removed the <Head> element to test something and forgot to add it again

Reply:
Thanks